### PR TITLE
change the viewport width

### PR DESCRIPTION
**Pull request by Aechelon-Online**
modified css for clean responsiveness

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -77,9 +77,6 @@
   "demo": "https://jsfiddle.net/allenluce/c49r5b4b/",
   "browsers": true,
   "badges": {
-    "list": [
-      "travisci",
-      "npmversion"
-    ]
+    "list": ["travisci", "npmversion"]
```
Comment (dougluce) [inline package.json:80]: Can you revert this formatting change?

**Review by dougluce — CHANGES_REQUESTED** [review]
Looks good for the most part, there's one small change I'm requesting.

File: package.json (hunk 1 above)
Comment (Aechelon-Online) [reply package.json:80]: For some reason, it gave me an error when it was formatted like what was in red. I'll try and put it back and see if it has no errors.

Comment (Aechelon-Online) [reply package.json:80]: I put it back, and it comes up with this error:
Incorrect type. Expected "array".

Comment (dougluce) [reply package.json:80]: Can you paste the command you're running and the error output?

**Pull request merged**